packages/E2ETest/windows/ReactUWPTestApp/App.xaml.cs

```diff
@@ -7,87 +7,53 @@
-    }
-
-    /// <summary>
-    /// Invoked when the application is launched normally by the end user.  Other entry points
-    /// will be used such as when the application is launched to open a specific file.
-    /// </summary>
-    /// <param name="e">Details about the launch request and process.</param>
-    protected override void OnLaunched(LaunchActivatedEventArgs e)
-    {
-      Frame rootFrame = Window.Current.Content as Frame;
-
-      // Do not repeat app initialization when the Window already has content,
-      // just ensure that the window is active
-      if (rootFrame == null)
-      {
-        // Create a Frame to act as the navigation context and navigate to the first page
-        rootFrame = new Frame();
-
-        rootFrame.NavigationFailed += OnNavigationFailed;
-
-        if (e.PreviousExecutionState == ApplicationExecutionState.Terminated)
+        /// <summary>
+        /// Initializes the singleton application object.  This is the first line of authored code
+        /// executed, and as such is the logical equivalent of main() or WinMain().
+        /// </summary>
+        public App()
         {
-          //TODO: Load state from previously suspended application
+            MainComponentName = "ReactUWPTestApp";
+
+#if BUNDLE
+            JavaScriptBundleFile = "index.windows";
+            InstanceSettings.UseWebDebugger = false;
+            InstanceSettings.UseLiveReload = false;
+#else
+            JavaScriptMainModuleName = "app/index";
+            InstanceSettings.UseWebDebugger = true;
+            InstanceSettings.UseLiveReload = true;
+#endif
+
+#if DEBUG
+            InstanceSettings.EnableDeveloperMenu = true;
+#else
+            InstanceSettings.EnableDeveloperMenu = false;
+#endif
+
+            PackageProviders.Add(new Microsoft.ReactNative.Managed.ReactPackageProvider()); // Includes any modules in this project
+            this.InitializeComponent();
         }
 
-        // Place the frame in the current Window
-        Window.Current.Content = rootFrame;
-      }
-
-      if (e.PrelaunchActivated == false)
-      {
-        if (rootFrame.Content == null)
+        /// <summary>
+        /// Invoked when the application is launched normally by the end user.  Other entry points
+        /// will be used such as when the application is launched to open a specific file.
+        /// </summary>
+        /// <param name="e">Details about the launch request and process.</param>
+        protected override void OnLaunched(LaunchActivatedEventArgs e)
         {
-          // When the navigation stack isn't restored navigate to the first page,
-          // configuring the new page by passing required information as a navigation
-          // parameter
-          rootFrame.Navigate(typeof(ReactTestPage), e.Arguments);
+            base.OnLaunched(e);
+            SystemNavigationManager.GetForCurrentView().AppViewBackButtonVisibility = AppViewBackButtonVisibility.Collapsed;
```

Comment:
>AppViewBackButtonVisibility [](start = 56, length = 27)

why do we need this? Is it a bug of ReactNative

Reply:
Yes, it is not implemented yet and will throw exception when user click on back button, see ReactInstanceManager::OnBackPressed

Reply:
I suggest you create a bug to track this.

Reply:
Spoke with Jon and created issue #3721  to track.